src/components/routing/routers/GameRouter.js

```diff
@@ -52,6 +53,11 @@ const GameRouter = props => {
           </GameGuard>
       </Route>
 
+        {/* rules */}
+        <Route exact path={`${props.base}/rules`}>
+            <Rules/>
+        </Route>
```

Comment:
the Routes were changed to be all in the GameGuard individually, but I'll probably do this directly in dev for Rules